src/components/transactions/TxType/styles.module.css

```diff
@@ -1,7 +1,7 @@
 .txType {
   display: flex;
   align-items: center;
-  flex-wrap: wrap;
+  flex-wrap: nowrap;
```

Comment:
Unrelated to notification, it fixes this:
<img width="305" alt="Screenshot 2023-10-02 at 15 18 13" src="https://github.com/safe-global/safe-wallet-web/assets/381895/46b01145-76f5-4a2d-9105-75651ee9eea8">